addon/doxycommentview/index.html

```diff
@@ -1,7 +1,7 @@
-<!DOCTYPE html>
-<html lang="en">
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "https://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US">
```

Comment:
Is there a reason to make the header use XHTML instead of HTML5? These are different standards and is probably also why the CDATA section is needed below. For XHTML also the file name should be .xhtml.

Reply:
- It is now the same as with the "normal" doxygen output (consistency)
- `CDATA` gave a small problem hence the `//` comment signs.